catalog.html

```diff
@@ -0,0 +1,160 @@
+      <li class="menu-catalog-navigation"><a href="#" class="button">Главная</li>
+      <li class="menu-catalog-navigation"><a href="#" class="button">Каталог товаров</li>
+      <li class="menu-catalog-navigation"><a href="#" class="button">Моноподы для селфи</li>
+      <section class="filter">
+      <form action="https://echo.htmlacademy.ru">
+          <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Стоимость,₽</legend>
+            от
+          <label><input class="conteol-input-min" type="number" name="min-price" value="0"></label>
+            до
+          <label><input class="conteol-input-min" type="number" name="max-price" value="5000"></label>
+          </fieldset>
+
+
+          <fieldset class="catalog-filter-color">
+            <legend class="catalog-filter-color">Цвет</legend>
+          <div>
+            <input type="checkbox" id="black" name="black" value="black" checked>
+            <label for="black">Черный</label>
+          </div>
+          <div>
+            <input type="checkbox" id="white" name="white" value="white" checked>
+            <label for="white">Белый</label>
+          </div>
+          <div>
+            <input type="checkbox" id="blue" name="blue" value="blue">
+            <label for="blue">Синий</label>
+          </div>
+          <div>
+            <input type="checkbox" id="red" name="red" value="red">
+            <label for="red">Красный</label>
+          </div>
+          <div>
+            <input type="checkbox" id="pink" name="pink value="pink">
+            <label for="pink">Розовый</label>
+          </div>
+        </fieldset>
+        <fieldset class="catalog-filter-radio">
+        <legend class="catalog-filter-radio">bluetooth</legend>
+          <div>
+            <input type="radio" id="yes" name="bluetooth" value="Есть" checked>
+            <label for="yes">Есть</label>
+          </div>
+          <input type="radio" id="no" name="bluetooth" value="Нет">
+            <label for="no">Нет</label>
+            <a href="#" class="button">Подписаться</a>
+          </div>
+          <div>
+            <input type="submit" id="show" name="show" value="Показать">
+            <label for="show">Показать</label>
+          </div>
+       </form>
+      </section>
+      <section class="sorting">
+      <form action="https://echo.htmlacademy.ru" method="POST">
+        <select size="2" multiple name="sorting" form="Сортировка">
+          <option value="^" selected>^</option>
+          <option value="v">V</option>
+        </select>
+        <div>
+          <input type="submit"  id="send" name="send" value="отправить">
+          <label for="send">Популярные</label>
+        </div>
+        </form>
+      </section>
+      <section>
+        <div>
+          <a class="logo-product" alt="logo-device" src="#" href="img"></a>
+          <a class="item_href" href="#">Любительская селфи палка</a>
+          <a class="price"><p>540P</p></a>
```

Comment:
Аналогично у остальных товаров.
```suggestion
          <p class="price">540P</p>
```